src/todo.js

```diff
@@ -1,5 +1,6 @@
+
 export function format(todo) {
-  return `${todo.id} - [${todo.done ? '✅': ' '}] ${todo.title}`;
+  return `${todo.id} - [${todo.done ? '✅' : ' '}] ${todo.title}`;
```

Comment:
Do we want x or ✅? I went with x

Reply:
x is fine to me, i saw, that you fixed that